test/e2e/e2e_suite_test.go

```diff
@@ -571,11 +522,3 @@ func getLatestSkuForMinor(version string, skus semver.Versions) string {
 	// otherwise, we just return the version as-is. This allows for versions in other formats, such as "latest" or "latest-1.21".
 	return version
 }
-
-// validateStableReleaseString validates the string format that declares "get be the latest stable release for this <Major>.<Minor>"
-// it should be called wherever we process a stable version string expression like "stable-1.22"
-func validateStableReleaseString(stableVersion string) (bool, []string) {
```

Comment:
why does this one need to move and not the other funcs above?

Reply:
it depends if the function is used outside one of the `_test.go` files.  In this case this function is by aks.go:https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/fb0a7f425c238eeeb6d1bd3ed16254f0884fb8d8/test/e2e/aks.go#L268

because the variable is defined in the `_test.go` the IDE compilers don't find it.  

I do find it interesting that when running `go test` that the variables defined in a `_test.go` are found by the standard `.go` files (which is why this doesn't through error when we run it in out suite.  :thinking:

Reply:
we should probably move the other functions above for resolving k8s versions to helpers anyways since they could technically be reused by other files, even if we're not using them now, wdyt?

Reply:
moved